src/_incydr_sdk/watchlists/models/responses.py

```diff
@@ -82,6 +82,24 @@ class WatchlistUser(ResponseModel):
     username: Optional[str] = Field(None, example="foo@bar.com")
 
 
+class WatchlistActor(ResponseModel):
+    """
+    A model representing a user whose associated with a watchlist.
+
+    **Fields**:
+
+    * **added_time**: `datetime` - The time the user was associated with the watchlist.
+    * **actor_id**: `str` - Unique actor ID.
+    * **actorname**: `str - Actor name.
```

Comment:
should this be `actor_name`?

Reply:
we could change it to actor_name for consistency. I'll do that.

Reply:
yeah, the example should be enough to make it obvious its an email